### ISSUE TITLE
Run in electron (or similar)

### ISSUE THREAD

**Issue by ferndot** [codexa/firetext#353]
This project has been essentially on hold (except for the awesome fork running at [Airborn OS](airbornos.com)) since Firefox OS for smartphones was cancelled.

I believe that with some work, it could be transformed into quite a useful app for the desktop.

The first stage of this is to get it running on desktop. Is anyone interested in getting the app to run in Electron or something similar?

I am available if you have any questions!

**Comment by parrajustin** [comment]
Hello joshua I was able to get firetext running in electron, with some modifications. 

![image](https://user-images.githubusercontent.com/1360989/31555969-4f4760f6-b000-11e7-8885-e4594fa8ba5f.png)

I'm still trying to fix a bug with the localization, but I was wondering how should I contribute this? Do I push it to a new branch?

**Comment by ferndot** [comment]
Hey @parrajustin this is great work 👍 

You can push the fix to a new branch and then submit a PR once it is ready. We adhere to the Git Flow development pattern.

Thanks!

**Comment by parrajustin** [comment]
@joshua-s for the PR what base branch do I use? On my fork I have an electron branch.

![image](https://user-images.githubusercontent.com/1360989/31562634-6a57397a-b019-11e7-9bef-2d15711da6e6.png)

**Comment by ferndot** [comment]
@parrajustin please use our develop branch :)

**Comment by ferndot** [comment]
This was fixed by @parrajustin!
